### ISSUE TITLE
PRIMARY_REGION is not considered on cluster initialization

### ISSUE THREAD

**Issue by davissp14** [fly-apps/postgres-ha#11]
Stolon has a keeper flag named `can-be-master`.  We use this in combination with the `PRIMARY_REGION` env var to restrict primary eligibility to the specified region.

The problem is that Stolon's Sentinel does not consider this flag on cluster initialization.  Instead, the primary keeper is chosen at random.  

More information can be found here: https://github.com/sorintlab/stolon/issues/840

**Workaround**

The recommended approach as of right now, is to provision your cluster within the primary region BEFORE spinning up cross-region read-replicas.  If you spin everything up at once, you will be at risk of an out-of-region member being assigned primary.

**Comment by mrkurt** [comment]
This is (currently) a non issue for `flyctl pg create` right? It just creates a cluster in a single region.

**Comment by davissp14** [comment]
@mrkurt Right, as far as provisioning goes this would only be an issue for manual deployments.   However, in the event the backend store is lost the cluster will be forced to re-initialize, which could open up another opportunity for this to bite us.
